### PR TITLE
Go rewrite copy comments script

### PR DESCRIPTION
**Pull request by zli82016**
**Release Note Template for Downstream PRs (will be copied)**

```release-note:none

```

### CODE REVIEW

**Comment by zli82016** [comment]
https://github.com/GoogleCloudPlatform/magic-modules/pull/11713 needs to be merged first before this PR.

File: mmv1/description-copy.go [hunk 1]
```diff
@@ -21,13 +21,262 @@ func CopyAllDescriptions() {
...
-	for i, id := range identifiers {
-		CopyText(id, len(identifiers)-1 == i)
+	for _, id := range identifiers {
+		CopyText(id)
 	}
+
+	copyComments()
+}
+
+// TODO rewrite: ServicePerimeters.yaml is an exeption and needs manually fixing the comments over after switchover
```
Comment (c2thorn) [inline mmv1/description-copy.go:31]: Will ServicePerimeters.yaml block future automatic refreshes or the comment just won't be copied over

**Review by c2thorn — APPROVED** [review]

File: mmv1/description-copy.go (hunk 1 above)
Comment (zli82016) [reply mmv1/description-copy.go:31]: No, ServicePerimeters.yaml  will not block the automatic refreshes.

The comments of ServicePerimeters.yaml  are copied to the Go yaml files. The copied comments are not accurate and need to be fixed after the switch over.

**Comment by zli82016** [comment]
Rebased the main branch to fix the conflicts and add more fields to renamedFields.

**Review by c2thorn — APPROVED** [review]

**Pull request merged**